frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/DemoGraphTableProvider.java

```diff
@@ -1,12 +1,16 @@
 package com.virtuslab.gitmachete.frontend.ui.impl.table;
 
 import com.intellij.ui.table.JBTable;
+import org.checkerframework.checker.guieffect.qual.UIEffect;
 
 import com.virtuslab.gitmachete.frontend.ui.api.table.IDemoGraphTableProvider;
 
 public class DemoGraphTableProvider implements IDemoGraphTableProvider {
   @Override
-  public JBTable getInstance() {
-    return DemoGraphTable.INSTANCE;
+  @UIEffect
+  public JBTable deriveInstance() {
+    // The reinstantiation is needed every time because without it
```

Comment:
Ok... so this was the reason behind #419?

Reply:
Actually, does restart of the IDE fix the problem? I had an impression that on your machine #419 persisted over IDE restarts

Reply:
Restart DO fix the problem to me

Reply:
Actually the same instance (as a field) was kept within `DemoGraphTable` and within `HelpAction`